arch/cpu/cc26xx-cc13xx/cc13xx-cc26xx-conf.h

```diff
@@ -81,10 +73,6 @@
 #if CC13XX_CONF_PROP_MODE
 #define NETSTACK_CONF_RADIO        prop_mode_driver
 
-#ifndef RF_CORE_CONF_CHANNEL
-#define RF_CORE_CONF_CHANNEL                      0
```

Comment:
I'm not sure about this one. Did we need the channel set to 0 here?

Reply:
Yes please, channels count from 0 upwards in .15.4g

Reply:
ok